documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -16,6 +16,7 @@ and the `bmc` `address` parameter for the `install-config.yaml` file.
 provisioning network. ({product-title} 4.4 only)
 | `hosts` |  | Details about bare metal hosts to use to build the cluster.
 | `defaultMachinePlatform` | | The default configuration used for machine pools without a platform configuration.
+| `disableCertificateVerification` | | This parameter is needed when we use `redfish` instead of `ipmi` to manage BMC addresses. The value should be `True` if you are using self-signed certificate for BMC addresses or `False` if you have a Certificate Authority. 
```

Comment:
This parameter is needed when using `redfish` or `redfish-virtualmedia` to manage BMC addresses. The value must be set to `True` when using self-signed certificate for BMC addresses.

Reply:
Added the same as suggested.